scss/lessonCard.scss

```diff
@@ -1,3 +1,6 @@
+.lesson-card__container {
```

Comment:
this is a state, for in progress. 

```
.lesson-card__container_inprogress
```

Look at `Modifier` -> boolean section.
https://en.bem.info/methodology/quick-start/#modifier